.coveragerc

```diff
@@ -0,0 +1,17 @@
+[run]
+branch = True
+
+[report]
+exclude_lines =
+    # Manual way to exclude blocks
```

Comment:
@Anaphory Could I maybe get a general introduction to the purpose/function of this file? It looks like an interface for other software...